lib/ldap/model/ad/group.rb

```diff
@@ -111,6 +114,10 @@ def type_id=(type)
       self['groupType'] = type.to_s
     end
 
+    def block_external?
+      self['extensionAttribute12'] == 'BLOCK_EXTERNAL'
+    end
+
```

Comment:
This should not be here, because it is IFAD-specific logic. This should go in our derived class. Important.